docs/agent/changelog.md

```diff
@@ -6,6 +6,11 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.16.0 - \[2024-09-09\]
+
+- Fix config upgrade failing with new v3 format
+- Improve agent commands for editing and validating the new v3 config format
+
```

Comment:
@russorat since there's probably more / different wording we'd like here.

Reply:
I might copy the changelog from v3.15.0 into this section, since it's unlikely anyone will look at the older version.

Reply:
Fair, I went back and forth on that. On it! 🙇